<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabletAreaSelectionView</class>
 <widget class="QWidget" name="TabletAreaSelectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>312</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="infoLayout">
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>The tablet area defines how the screen space is mapped to your tablet. For each screen a different tablet area can be selected. </string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="controlsLayout">
       <item>
        <widget class="QGroupBox" name="screenGroupBox">
         <property name="title">
          <string>Screen</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="screenCenterLayout">
            <item>
             <widget class="Wacom::AreaSelectionWidget" name="screenArea" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="screenToggleButton">
            <property name="toolTip">
             <string>Toggle between fullscreen and each single screen.</string>
            </property>
            <property name="text">
             <string>Toggle Screen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="controlsGroupBox">
         <property name="toolTip">
          <string>Select the valid tablet area used for the selected screen space.</string>
         </property>
         <property name="title">
          <string>Map Screen To</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="fullTabletRadioButton">
            <property name="text">
             <string>&amp;Full Tablet</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="tabletAreaRadioButton">
            <property name="text">
             <string>&amp;Tablet Area</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forceProportionsButton">
            <property name="toolTip">
             <string>Set the tablet area proportional to the selected screen area.</string>
            </property>
            <property name="text">
             <string>Set Screen Proportions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="calibrateButton">
            <property name="text">
             <string>Calibrate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="controlsFreeSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="tabletGroupBox">
       <property name="title">
        <string>Tablet</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="areaCenterLayout">
          <item>
           <widget class="Wacom::AreaSelectionWidget" name="areaWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="warningLayout">
          <item>
           <widget class="QLabel" name="warningIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="warningLabel">
            <property name="text">
             <string>This mapping can only be applied in absolute tracking mode.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonLockProportions">
          <property name="text">
           <string>Lock proportions</string>
          </property>
          <property name="icon">
           <iconset theme="lock"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="fineTuneGroupBox">
          <property name="title">
           <string>Tablet area fine tuning</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEditWidth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Width</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelHorizontalTune">
             <property name="text">
              <string>Horizontal</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Left margin</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelVerticalTune">
             <property name="text">
              <string>Vertical</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Top margin</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEditHeight">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Height</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="tabletFreeSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Wacom::AreaSelectionWidget</class>
   <extends>QWidget</extends>
   <header>areaselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>calibrateButton</sender>
   <signal>clicked()</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onCalibrateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forceProportionsButton</sender>
   <signal>clicked()</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onForceProportionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullTabletRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onFullTabletSelected(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabletAreaRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onTabletAreaSelected(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screenToggleButton</sender>
   <signal>clicked()</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onScreenToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonLockProportions</sender>
   <signal>toggled(bool)</signal>
   <receiver>TabletAreaSelectionView</receiver>
   <slot>onLockProportionsToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCalibrateClicked()</slot>
  <slot>onForceProportionsClicked()</slot>
  <slot>onFullTabletSelected(bool)</slot>
  <slot>onTabletAreaSelected(bool)</slot>
  <slot>onScreenToggle()</slot>
  <slot>onLockProportionsToggled(bool)</slot>
 </slots>
</ui>
